### PR TITLE
[DOCS] fixed navbar styling

### LINKED ISSUE #11611: [Docs] Navbar not responsive and UI glitches
### Current State
PHOTO-2024-08-11-01-26-14  
PHOTO-2024-08-11-01-17-47

Navbar height is overflowing and the first text is misaligned to the left. Adjustments needed to ensure proper alignment and responsive behavior.
### Desired State

PHOTO-2024-08-11-01-28-28
PHOTO-2024-08-11-01-17-12

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by thisisharsh7**
**Notes for Reviewers**
# Kindly review it.

This PR fixes #11611

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 902ba9b211cf8ed7d38a35d57687d0e07361bba3 at: https://66b7cc8118f5a5926487cc9f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 6fb4d55f3323265c07ad523472695cefd71b8596 at: https://66b7d132fdff58935e3d43eb--meshery-docs-preview.netlify.app

**Comment by hargunkaur286** [comment]
@thisisharsh7 Please wait for an issue to get validated and assigned to you before making a PR

**Comment by thisisharsh7** [comment]
> @thisisharsh7 Please wait for an issue to get validated and assigned to you before making a PR

Thank you for the note. I apologise for not waiting for the issue to be validated and assigned. I’ll make sure to follow the process correctly in the future. If there’s anything that needs to be adjusted in the current PR, please let me know.

**Comment by sudhanshutech** [comment]
@thisisharsh7 please add this topic to discuss to today website meeting here as agenda: https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit. 
Its today at 5:50pm IST

File: docs/_sass/navbar.scss
Comment (leecalcote) [inline docs/_sass/navbar.scss:1]: Are there any changes included in this file that are material (any that _actually_ need to be changed)? If not, can we exclude this file from the PR?

File: docs/_sass/elements.scss
Comment (leecalcote) [inline docs/_sass/elements.scss:1]: Thanks for contributing, @thisisharsh7 👍 👏

File: docs/_sass/navbar.scss
Comment (jameshorton2337) [reply docs/_sass/navbar.scss:1]: @thisisharsh7, do you have an answer to this feedback?

**Review by Ashparshp — COMMENTED** [review]
LGTM, but please avoid any unnecessary changes like spaces, tabs, or anything that isn't a functional change, as it makes the review process harder. Anyway, thank you @thisisharsh7!

**Comment by Ashparshp** [comment]
@thisisharsh7, any updates?

**Comment by Ashparshp** [comment]
@thisisharsh7. checking last time?

**Comment by thisisharsh7** [comment]
@Ashparshp  I've already done with changes, so I cannot add anything now. Thank you for the note as well. In the future, I will keep in mind and not push changes like spaces, tabs, or any other changes that are not functional. Apologise.

**Pull request merged**